{"levelname": "ERROR", "asctime": "2025-08-04T20:25:09.004Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 81, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "e1b52c82-8307-4c8b-a53c-042d78e5e634", "collection_version_id": "27ee1007-9425-4f03-a035-c849384c0f9d", "errors": ["error message"]}
